CMakeLists.txt

```diff
@@ -55,3 +55,8 @@ set_property(TARGET depthai_ctrl PROPERTY CXX_STANDARD_REQUIRED ON)
 set_property(TARGET depthai_ctrl PROPERTY CXX_EXTENSIONS OFF)
 
 install(TARGETS depthai_ctrl)
+
+install(FILES
```

Comment:
Do we need to verify that the permissions are ok for the 80-movidius.rules file?

Reply:
Done